tests/capi/trajectory.cpp

```diff
@@ -301,19 +301,20 @@ TEST_CASE("Write trajectory to memory") {
 
     CHECK_STATUS(chfl_trajectory_write(trajectory, frame));
 
-    const char* result = nullptr;
-    uint64_t size_of_result;
-    CHECK_STATUS(chfl_trajectory_memory_buffer(trajectory, &result, &size_of_result));
-    CHECK(std::string(result, size_of_result) == EXPECTED_CONTENT);
+    const char* data = nullptr;
+    uint64_t size;
+    CHECK_STATUS(chfl_trajectory_memory_buffer(trajectory, &data, &size));
+    CHECK(size == std::strlen(EXPECTED_CONTENT));
```

Comment:
I think we need `#include <cstring>` for `strlen`.

Reply:
Should be good now!